src/estimation/src/BerdyHelper.cpp

```diff
@@ -1366,6 +1387,48 @@ bool BerdyHelper::computeBerdySensorMatrices(SparseMatrix<iDynTree::ColumnMajor>
         // bY for the joint wrenches is zero
     }
 
+    ////////////////////////////////////////////////////////////////////////
+    ///// COM ACCELERATION SENSOR
+    ////////////////////////////////////////////////////////////////////////
+    if (m_options.includeCoMAccelerometerAsSensor)
+    {
+
+        // Get the row index corresponding to the com accelerometer sensor
+        // TODO: Correct the link index pssed here
+        IndexRange comAccelerometerRange = this->getRangeLinkSensorVariable(COM_ACCELEROMETER_SENSOR, 0);
```

Comment:
This will be changed as a consequence of https://github.com/robotology/idyntree/pull/571/files#r320051199

Reply:
Updated in https://github.com/robotology/idyntree/pull/571/commits/2f12467492a2ba8216d1ced0ad4ba8531e7f49a3